components/forms/ltft/LtftSummary.tsx

```diff
@@ -45,43 +46,51 @@ const LtftSummary = ({
   const columnHelper = createColumnHelper<LtftSummaryObj>();
 
   // Header
-  const renderNameHeader = ({ column }) => (
+  const renderNameHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
       title="Name"
       data-cy={`table-column_${column.id}`}
     />
   );
-  const renderCreatedHeader = ({ column }) => (
+  const renderCreatedHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
-      title="Created date"
+      title="Created"
       data-cy={`table-column_${column.id}`}
     />
   );
-  const renderStatusHeader = ({ column }) => (
+  const renderLastModifiedHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
-      title="Status"
+      title="Last modified"
       data-cy={`table-column_${column.id}`}
     />
   );
-  const renderLastModifiedHeader = ({ column }) => (
+  const renderStatusHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
-      title="Status date"
+      title="Status"
       data-cy={`table-column_${column.id}`}
     />
   );
 
   //Header value
-  const renderValue = props => <span>{props.renderValue()}</span>;
-  const renderDayValue = props => (
+  const renderValue = (props: any) => <span>{props.renderValue()}</span>;
```

Comment:
We might be able to avoid using the `any` Type here (always a last resort! 😅).
e.g. `props: { renderValue: () => ReactNode }` should work.